packages/block-library/src/image/image.js

```diff
@@ -394,6 +397,17 @@ export default function Image( {
 		</>
 	);
 
+	const captionControls = (
+		<Fill name="RichText.ToolbarControls.text-color">
```

Comment:
I'm using a reserved slot for `text-color,` but we should probably add a custom one for `text-alignment.`